apps/services/contentful-entry-tagger/infra/contentful-entry-tagger-service.ts

```diff
@@ -0,0 +1,21 @@
+import { service, ServiceBuilder } from '../../../../infra/src/dsl/dsl'
+
+export const serviceSetup = (): ServiceBuilder<'contentful-entry-tagger-service'> =>
+  service('contentful-entry-tagger-service')
```

Comment:
This name has to match the one in the [helm repo](https://github.com/island-is/helm/pull/645).

Reply:
I changed the name in the helm repo to be contentful-entry-tagger-service